backlog/migrations/0076_auto_20210729_1509.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-07-29 15:09
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('backlog', '0075_auto_20210723_1404'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='assemblyannotationjob',
+            name='protein_db',
+        ),
+        migrations.AlterField(
```

Comment:
This migration statement is not present in the changes you made to the models. Maybe you forgot to commit a file?